src/index.html

```diff
@@ -1,13 +1,65 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+
+      <div class="stars stars--0">
```

Comment:
```suggestion
    <div class="container">
      <div class="stars stars--0">
```

Reply:
Don't add empty line between parent and children, only between siblings

Reply:
Sorry, I did the task before the start of the course, so the mistake repeated itself, I remembered this rule.